scripts/reactor_deployment.py

```diff
@@ -328,12 +384,19 @@ def rand_deployment(df, base_col, ar_dict,
     -------
     df: pandas dataframe
         The dataframe of capacity information with the deployed reactors.
+
+    Notes
+    -----
+    * This function assumes that there is a column called 'Year' that is not
+    the column of indices.
     """
 
     # initialize the number of reactor columns
     df = reactor_columns(df, ar_dict)
 
-    for year in range(len(df[base_col])):
+    start_index = df[df['Year'] == dep_start_year].index[0]
+
+    for year in range(start_index, len(df[base_col])):
```

Comment:
Right... this pattern is so far identical to earlier implementations. For me, it would make more sense for the API to have a single function to update the capacity in a new column and have users pass a string parameter (e.g., "greedy" or "linear" or "random") and have the capacity_update function call each function from a dictionary. I can elaborate if needed.